frontend/src/components/elements/CodeBlock/CodeBlock.scss

```diff
@@ -21,6 +21,10 @@
   See https://prismjs.com/download.html#themes=prism&languages=markup+css+clike+javascript
 */
 
+div.code-block {
+  margin: 0 0 1rem 0;
+}
```

Comment:
Why is this needed?

Reply:
the snapshot was failing due to this, because the div has no margin